media-sound/rhythmbox/rhythmbox-3.4.1.ebuild

```diff
@@ -45,7 +45,10 @@ COMMON_DEPEND="
 	gnome-keyring? ( >=app-crypt/libsecret-0.18 )
 	libnotify? ( >=x11-libs/libnotify-0.7.0 )
 	lirc? ( app-misc/lirc )
-	python? ( >=dev-python/pygobject-3.0:3[${PYTHON_USEDEP}] )
+	python? (
+		>=dev-libs/libpeas-0.7.3[python,${PYTHON_USEDEP}]
```

Comment:
Actually this one can stay in RDEPEND as libpeas does not provide a pkgconfig for python support so it is not used at build time.